App programmering/Opgave/app/src/main/java/com/example/app/Settings.java

```diff
@@ -0,0 +1,57 @@
+package com.example.app;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.content.pm.ActivityInfo;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+public class Settings extends AppCompatActivity {
+
+    public static final String MIME_VIGO6 = "application/x.proces-data.vigo6_nfc";
+    private boolean convert = false;
+    private Button ButtonInitialize;
+    private EditText SerialNumber;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_settings);
+        lockScreenOrientation();
+
+        SerialNumber = (EditText) findViewById(R.id.editTextSerialnumberSettings);
+        ButtonInitialize = (Button) findViewById(R.id.buttonInitialize);
```

Comment:
Behøver du disse casts?

Reply:
Det er for at få oprettet en connection til de henholdvise ting, så ja

Reply:
Rettet

Reply:
Fandt ud af de faktisk var nødvendige  fordi de bliver gjort brug af over i min main aktivitet, så nu da jeg fjernede dem crashede den hver gang jeg åbnede settings activityen. Det er i forhold til de intents jeg bruger for at sende frem og tilbage